services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/client/grpc/DataPlaneClientImpl.java

```diff
@@ -112,13 +112,13 @@ public DataPlaneClientImpl(Config globalConfig) {
         //Handle all failed hosts
         return futures.parallelStream().filter(Objects::nonNull).map(future -> {
             try {
-                return future.get().getHostIp();
+                return future.get();
             } catch (InterruptedException | ExecutionException e) {
                 e.printStackTrace();
             }
 
             return null;
-        }).collect(Collectors.toList());
+        }).filter(item -> item != null).collect(Collectors.toList());
```

Comment:
@DavidLiu506 if possible please add a UT to test this change to see if this filter works.

Reply:
It works, if not there will be a DMP failed response.